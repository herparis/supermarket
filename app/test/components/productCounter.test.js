import React from 'react'
import { shallow } from 'enzyme'

import ProductCounter from '../../src/components/productCounter/productCounter'

function checkRender(wrapper) {
  expect(wrapper.isEmptyRender()).toBe(false)
  expect(wrapper).toMatchSnapshot()
}

describe('ProductCounter components rendeding', () => {
  const productCounter = shallow(<ProductCounter />)
  it('ProductCounter renders', () => {
    checkRender(productCounter)
  })
})