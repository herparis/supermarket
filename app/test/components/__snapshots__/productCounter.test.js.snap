// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProductCounter components rendeding ProductCounter renders 1`] = `ShallowWrapper {}`;
